src/main/kotlin/pt/up/fe/ni/website/backend/model/Post.kt

```diff
@@ -0,0 +1,16 @@
+package pt.up.fe.ni.website.backend.model
+
+import java.util.Date
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+class Post(
+    var title: String,
+    var body: String,
+    var href: String,
+    var templatePath: String,
+    var publishDate: Date,
```

Comment:
```suggestion
    val publishDate: Date,
```

Reply:
I think it might also be worth it to create a "lastUpdatedAt" date. Wdyt @BrunoRosendo @AvilaAndre?

Reply:
I think it might also be worth it to create a "lastUpdatedAt" date. Wdyt @BrunoRosendo @AvilaAndre?

Reply:
I agree. Actually, publishDate and lastUpdatedAt should probably be automatically created by the BE, right? I completely missed that

Reply:
Check this thread https://stackoverflow.com/questions/51709727/spring-boot-jpacreateddate-lastmodifieddate-not-being-populated-when-saving-th

Reply:
@AvilaAndre Maybe this post helps more https://self-learning-java-tutorial.blogspot.com/2019/08/spring-data-jpa-createddate.html

I think you can try adding @CreatedDate to the publishDate and create a lastUpdatedAt with @LastModifiedDate. This will hopefully do everything automatically but please test it. Also test if it's not possible to create a post with an arbitrary publishDate (it shouldn't be possible)

Reply:
It worked! It isn't possible to create a Post with an arbitrary date, I think it is because it changes the publishDate before inserting it into the repository so it ends up changing the arbitrary value anyways. Updating it is also impossible as there is no publishDate on the PostDto. (I tested it)